ogcapi-processes/echo.http

```diff
@@ -0,0 +1,342 @@
+###
```

Comment:
Is this intended to be tracked? If yes maybe move to a log / test or output directory.